<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OGL</class>
 <widget class="QMainWindow" name="OGL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OGL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="RendererWidget" name="openGLWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="Param_GroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>1280</width>
      <height>440</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="ModelParam_GroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>180</width>
       <height>220</height>
      </rect>
     </property>
     <property name="title">
      <string>Model</string>
     </property>
     <widget class="QDoubleSpinBox" name="Model_AngleX_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="Model_AngleX_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>AngleX</string>
      </property>
     </widget>
     <widget class="QLabel" name="Model_AngleY_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>AngleY</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Model_AngleY_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="Model_AngleZ_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>AngleZ</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Model_AngleZ_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="Model_TY_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>TY</string>
      </property>
     </widget>
     <widget class="QLabel" name="Model_TZ_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>TZ</string>
      </property>
     </widget>
     <widget class="QLabel" name="Model_TX_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>TX</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Model_TX_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Model_TY_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Model_TZ_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="CameraParam_GroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>180</width>
       <height>220</height>
      </rect>
     </property>
     <property name="title">
      <string>Camera</string>
     </property>
     <widget class="QLabel" name="Camera_AngleY_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>AngleY</string>
      </property>
     </widget>
     <widget class="QLabel" name="Camera_TX_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>TX</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Camera_TZ_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="Camera_AngleZ_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>AngleZ</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Camera_TY_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="Camera_TY_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>TY</string>
      </property>
     </widget>
     <widget class="QLabel" name="Camera_AngleX_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>AngleX</string>
      </property>
     </widget>
     <widget class="QLabel" name="Camera_TZ_Label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>TZ</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Camera_AngleX_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Camera_AngleY_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Camera_AngleZ_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="Camera_TX_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="CameraMatrix_Label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CameraMatrix</string>
     </property>
    </widget>
    <widget class="QLabel" name="CameraMatrix_Value_Label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CameraAngleXMatrix_Label">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>AngleXMatrix</string>
     </property>
    </widget>
    <widget class="QLabel" name="CameraAngleXMatrix_Value_Label">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>60</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CameraAngleYMatrix_Label">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>AngleYMatrix</string>
     </property>
    </widget>
    <widget class="QLabel" name="CameraAngleZMatrix_Label">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>AngleZMatrix</string>
     </property>
    </widget>
    <widget class="QLabel" name="CameraAngleYMatrix_Value_Label">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>60</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CameraAngleZMatrix_Value_Label">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>60</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CameraTMatrix_Label">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TMatrix</string>
     </property>
    </widget>
    <widget class="QLabel" name="CameraTMatrix_Value_Label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RendererWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>RendererWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
